builddefs/common_features.mk

```diff
@@ -120,7 +120,7 @@ ifeq ($(strip $(MOUSEKEY_ENABLE)), yes)
     MOUSE_ENABLE := yes
 endif
 
-VALID_POINTING_DEVICE_DRIVER_TYPES := adns5050 adns9800 analog_joystick azoteq_iqs5xx cirque_pinnacle_i2c cirque_pinnacle_spi paw3204 pmw3320 pmw3360 pmw3389 pimoroni_trackball custom
+VALID_POINTING_DEVICE_DRIVER_TYPES := adns5050 adns9800 analog_joystick azoteq_iqs5xx cirque_pinnacle_i2c cirque_pinnacle_spi paw3204 pmw3320 pmw3360 pmw3389 pimoroni_trackball ps2_mouse custom
```

Comment:
This may/does cause issues since line 134-135 expects a file: 
```make
        ifneq ($(strip $(POINTING_DEVICE_DRIVER)), custom)
            SRC += drivers/sensors/$(strip $(POINTING_DEVICE_DRIVER)).c
```
So it tries to compile `drivers/sensors/ps2_mouse.c`, and errors out during linking, for me.  At the very least, this needs a stub file so that it will compile.  Or to also check for ps2_mouse, in addition to custom (or to check if the file even exists)

Reply:
Good catch, moved `ps2_mouse.[ch] `to `drivers/sensors/`